docs/antora.yml

```diff
@@ -0,0 +1,6 @@
+name: Registry 
+version: 2.21.0-SNAPSHOT
```

Comment:
❔ I released Registry 2.22.0 just to get the next version out there. I currently have master at 2.22.1-SNAPSHOT. Thoughts?

Reply:
I haven't yet figured out the best way to keep them in sync, but I'm looking into it